libs/agno/agno/agent/agent.py

```diff
@@ -2355,6 +2355,9 @@ def load_agent_session(self, session: AgentSession):
             if isinstance(self.memory, dict) and "create_user_memories" in self.memory:
                 # Convert dict to AgentMemory
                 self.memory = AgentMemory(**self.memory)
+                # Convert dict to Memory
+            elif isinstance(self.memory, dict):
+                self.memory = Memory(**self.memory)
```

Comment:
This won't work if `runs` is in the dictionary, since that is not in the initializer. I think we need to make a `from_dict` method or just instantiate it blank since it will be populated below.

Reply:
Pushed a fix to address